pkg/vm/memory/memory.go

```diff
@@ -150,6 +150,15 @@ func (m *Memory) GetFelt(addr Relocatable) (lambdaworks.Felt, error) {
 	return lambdaworks.FeltZero(), err
 }
 
+// Get a range of memory from the starting relocatable to the starting relocatable + size
+func (m *Memory) GetRange(start Relocatable, size uint) []MaybeRelocatable {
+	var res []MaybeRelocatable
+	for i := uint(0); i < size; i++ {
+		res = append(res, m.Data[start.AddUint(i)])
```

Comment:
Use `Memory.Get` here, fetching from the map directly will yield the zero value if the key is not present, creating a nil `MaybeRelocatable` in the process

Reply:
Resolved: https://github.com/lambdaclass/cairo-vm_in_go/pull/241/commits/67a34851ac08e02fa7c8583fd09fa32eefe50c64